main.go

```diff
@@ -60,6 +62,33 @@ func (m metrics) IncErrorIncursions() {
 	m.ErrorIncursions.Inc()
 }
 
+func MetricsEndpointBasicAuthEnabled() bool {
```

Comment:
This function is introduced twice, use the one in the `utils` package

Reply:
Removed duplicated code